packages/angular/src/generators/utils/angular-version-utils.ts

```diff
@@ -45,3 +45,15 @@ export function getInstalledAngularVersionInfo(tree: Tree) {
     major: major(installedVersion),
   };
 }
+
+export function getInstalledPackageVersionInfo(tree: Tree, pkgName: string) {
```

Comment:
Let's rename this file to `version-utils.ts`, so the name is not tied just to Angular utils.